src/content/docs/en/guides/integrations-guide/svelte.mdx

```diff
@@ -66,17 +66,17 @@ Most package managers will install associated peer dependencies as well. If you
 <PackageManagerTabs>
   <Fragment slot="npm">
   ```sh
-  npm install svelte
+  npm install svelte typescript
```

Comment:
Should the sentence above also match something like what you added for React? "you'll need to install Svelte with typescript" if you're making this change?